### PR TITLE
[PUB-3053] Update direct posting modal with new link and copy

### PR DESCRIPTION
**Pull request by albennett**
## Description
- Update direct posting modal with a new link and more extensive instructions. 
- Refactor with react-i18next translations
- Fix button labels to Title Case
https://buffer.atlassian.net/browse/PUB-3053

## Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have tested different user stories. 
-   [ ] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/i18n/translations/en-us.json [hunk 1]
```diff
@@ -101,12 +101,13 @@
   },
   "instagram-direct-posting-modal": {
     "cta1": "Yes! Let's do it!",
-    "cta2": "I've Converted It to Business",
-    "description": "Direct Scheduling allows you to post images and videos to Instagram directly from Buffer, just as you can with any other network.",
+    "cta2": "I've converted to business",
```
Comment (anafilipadealmeida) [inline packages/i18n/translations/en-us.json:104]: I remember at some point marketing wanted us to have Title Case in all CTAs. Is this no longer the case?

File: packages/ig-direct-posting-modal/components/InstagramDirectPostingModal/index.jsx [hunk 2]
```diff
@@ -29,26 +28,25 @@ const InstagramDirectPostingModal = ({
...
-          <div style={textWrapperStyle}>
-            <Text type="p">{translations.description}</Text>
-          </div>
+          <Text type="p">{translations.description}</Text>
           {!isBusinessOnInstagram && (
-            <div style={textWrapperStyle}>
+            <>
               <Text type="p">
-                {translations.instructions}
+                {translations.instructions1}
```
Comment (anafilipadealmeida) [inline packages/ig-direct-posting-modal/components/InstagramDirectPostingModal/index.jsx:40]: Could be cool at some point to implement the translation methods from react-i18next that Maya brought

**Review by anafilipadealmeida — APPROVED** [review]
Looks good to me! 🙂 Just left a question about marketing!

File: packages/ig-direct-posting-modal/components/InstagramDirectPostingModal/index.jsx (hunk 2 above)
Comment (albennett) [reply packages/ig-direct-posting-modal/components/InstagramDirectPostingModal/index.jsx:40]: Hey, I considered refactoring with it, however, I was thinking we'd be able to remove this direct posting package in October (the new oauth forces users to set up direct scheduling). However, now with more thought, we may still need the direct posting flow if a user removes their fb page 🤔  

I'll add a change, I get too excited at a chance to refactor 😂 😂

File: packages/i18n/translations/en-us.json (hunk 1 above)
Comment (albennett) [reply packages/i18n/translations/en-us.json:104]: Ooh, thanks for letting me know! I confirmed with Anne on the Title Case, as I too wasn't sure of the standards. I'll confirm again with Anne, as I'm unsure if she's aware of the marketing context. Do you know if it's in notion or anything? 🤔

Comment (anafilipadealmeida) [reply packages/i18n/translations/en-us.json:104]: I just realized it's also in the BDS guide: https://bufferapp.github.io/ui/#/ui/Guides/buttons. 'Do: Use title case for button labels, e.g. "Add to Queue"'. The initial chat I had one year ago on Slack was this: https://buffer.slack.com/archives/C151RRDL1/p1561107718274100

Comment (anafilipadealmeida) [reply packages/i18n/translations/en-us.json:104]: Right now it seems the only button not following that Title Case rule in Publish is the one on the deprecation of personal IG accounts banner

Comment (albennett) [reply packages/i18n/translations/en-us.json:104]: Thanks! 🙌  I need to get in the habit of checking the guides more. I'll share how to get to the guides with Anne, too. 😄

**Pull request merged**